tools/coverage/coverage.py

```diff
@@ -834,7 +834,7 @@ def _digest(self, lineLogType, lineLog):
 
 
     def parsePFWlog(self, lines):
-        for lineNb, lineLog in enumerate(lines):
+        for lineIndex, lineLog in enumerate(lines):
```

Comment:
The offset may be given here:

    enumerate(lines, 1)
    # or
    enumerate(lines, start=1)

see https://docs.python.org/3/library/functions.html#enumerate